text/0006-architecture-review-process.md

```diff
@@ -143,13 +141,11 @@ A project under review will be in one of the following states:
 
 Generally our review process focuses on projects in the “Roadmap” or “Design” stages. However a project in the “Execution” or “Landing” stage may need subsequent review to decide if the project is being done the correct way, or if it should be continued at all, so it may return to either of the “Roadmap” or “Design” phases.
 
-After the review, the reviewer and responsible decision makers should provide concrete recommendations and examples for the next teams to undergo the review process. The results should be disseminated as widely as possible, with the package documents and recommendation document archived. Some version of the decision should be publicly linked, so that it can be the decision of record, to which blog posts, mailing list posts, the message saying that the GitHub repository is obsolete, etc., can refer. The goal is to avoid critical technical and resourcing decisions being internally settled and only — later, begrudgingly — being made public on some mailing list. Or, per RACI, the goal is to broaden the set of those Informed.
+After the review, the reviewer and responsible decision makers should provide concrete recommendations and examples for the next teams to undergo the review process. The results should be disseminated as widely as possible, with the package documents and recommendation document archived. Some version of the decision should be publicly linked, so that it can be the decision of record, to which blog posts, mailing list posts, the message saying that the GitHub repository is obsolete, etc., can refer. The goal is to avoid critical technical and resourcing decisions being internally settled and only — later, begrudgingly — being made public on some mailing list.
 
 ## Examples
 
-The following reviews have been completed and can be used as a template for a successful architectural review.
-
-* [XBL Removal](https://mozilla.github.io/firefox-browser-architecture/text/0007-xbl-design-review-packet.html)
+A number of reviews have been completed and can be used as a template for a successful architectural review. [XBL Removal](0007-xbl-design-review-packet.md) was the first example of a design review, [Sync and Storage](0008-sync-and-storage-review-packet.md) was the first roadmap review, however any of the documents with a title that includes "review" in [the text folder](..) could be used.
```

Comment:
Nit: "the text folder" seems like a confusing reference for an average reader of this document on the website, given that one can't actually load the text/ directory of the site (loading https://mozilla.github.io/firefox-browser-architecture/text/ returns a "page not found" error), and the link actually leads to the index file for the top-level directory of the site (https://mozilla.github.io/firefox-browser-architecture/), which makes no mention of a "text folder."

Instead of `any of the documents with a title that includes "review" in [the text folder](..) could be used`, I recommend something like `any of the [documents on this website](..) with a title that includes "review" could be used.`

Reply:
Fixed in a13378a.